src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -966,3 +966,124 @@
     text: >
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
+
+helps['acr scope-map'] = """
+type: group
+short-summary: Manage scope access maps for Azure Container Registries.
+"""
+
+helps['acr scope-map create'] = """
+type: command
+short-summary: Create a scope map for an Azure Container Registry.
+examples:
+  - name: Create a scope map that allows content/write and metadata/read actions for `hello-world` repository, and content/read action for `hello-world-again`.
+    text: >
+        az acr scope-map create -n MyScopeMap -r MyRegistry --add hello-world content/write metadata/read --add hello-world-again content/read --description "Sample scope map."
+"""
+
+helps['acr scope-map delete'] = """
+type: command
+short-summary: Delete a scope map under an Azure Container Registry.
+examples:
+  - name: Delete the scope map 'MyScopeMap'.
+    text: >
+        az acr scope-map delete -r MyRegistry -n MyScopeMap
```

Comment:
Add -n MyScopeMap as the first parameter to keep it consistent with the create example.

Reply:
Done.